x-pack/plugins/ml/public/alerting/severity_control/severity_control.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiFormRow, EuiRange, EuiRangeProps } from '@elastic/eui';
+import { SEVERITY_OPTIONS } from '../../application/components/controls/select_severity/select_severity';
+import { ANOMALY_THRESHOLD } from '../../../common';
+import './styles.scss';
+
+export interface SeveritySelectorProps {
+  value: number | undefined;
+  onChange: (value: number) => void;
+}
+
+const MAX_ANOMALY_SCORE = 100;
+
+export const SeverityControl: FC<SeveritySelectorProps> = React.memo(({ value, onChange }) => {
+  const levels: EuiRangeProps['levels'] = [
+    {
+      min: ANOMALY_THRESHOLD.LOW,
+      max: ANOMALY_THRESHOLD.MINOR - 1,
+      color: 'success',
+    },
+    {
+      min: ANOMALY_THRESHOLD.MINOR,
+      max: ANOMALY_THRESHOLD.MAJOR - 1,
+      color: 'primary',
+    },
+    {
+      min: ANOMALY_THRESHOLD.MAJOR,
+      max: ANOMALY_THRESHOLD.CRITICAL,
+      color: 'warning',
+    },
+    {
+      min: ANOMALY_THRESHOLD.CRITICAL,
+      max: MAX_ANOMALY_SCORE,
+      color: 'danger',
+    },
+  ];
+
+  const toggleButtons = SEVERITY_OPTIONS.map((v) => ({
+    value: v.val,
+    label: v.display,
+  }));
+
+  return (
+    <EuiFormRow
+      fullWidth
+      label={
+        <FormattedMessage
+          id="xpack.ml.severitySelector.formControlLabel"
+          defaultMessage="Select severity threshold"
+        />
+      }
+    >
+      <>
```

Comment:
Since there's only EuiRange here, I don't think we need the fragment here.

Reply:
Removed in 080565c8d7499bf8357e3192ba6a9e59e900a9b9